src/app/guidesite/page.tsx

```diff
@@ -0,0 +1,363 @@
+    'link',
+    `<a href="${linkUrl}" class="text-blue-500 underline" target="_blank" rel="noopener noreferrer">${linkText}</a>`
+  );
+
+  return (
+    <div>
+      <h2 className="text-bold font-bold mb-4 pt-10">{title}</h2>
+      <p className="text-main mb-4 ml-10">
+        <span className="font-bold block">Registrar&apos;s Office:</span> <br />
+        <span
+          dangerouslySetInnerHTML={{
+            __html: mainTextWithLink,
+          }}
+        />
+      </p>
+
+      {/* Main Content List */}
+      <ul className="list-decimal list-outside ml-24">
+        {content.map((document, index) => (
+          <li key={index} className="" style={{ fontWeight: 'bold' }}>
+            <span className="font-bold">{document.item}</span>
+            {document.details && (
+              <>
+                {' '}
+                -- <span className="font-normal">{document.details}</span>
+              </>
+            )}
+          </li>
+        ))}
+      </ul>
+
+      <div className="mt-[15px] ml-[70px]">
+        <Button text="Request Registrar's Certificate" />
+      </div>
+
+      <div className="mt-8">
+        <ul className="list-decimal list-outside ml-10">
+          <span className="font-bold block">
+            Office of the Student Affairs:
+          </span>
+          {OSA.map((document, index) => (
+            <li
+              key={index}
+              className="mb-2 ml-5"
+              style={{ fontWeight: 'bold' }}
+            >
+              <span className="font-normal">{document.item}</span>
+
+              <ul className="list-[lower-alpha] list-outside ml-6 font-bold">
+                <li className="font-bold">
+                  <span className="font-normal">
+                    {document.details.mainText.split('link')[0]}
+                    <Link
+                      href={document.details.linkUrl}
+                      className="underline text-blue"
+                    >
+                      {document.details.linkText}
+                    </Link>
+                    {document.details.mainText.split('link')[1]}
+                  </span>
+                </li>
+              </ul>
+            </li>
+          ))}
+        </ul>
+
+        <div className="mt-[15px] ml-[70px]">
+          <Button text="Request Certificate" />
+        </div>
+      </div>
+    </div>
+  );
+};
+const Guidesite = () => {
+  const { title, steps } = guidesiteData.venueReservations;
+  return (
+    <>
+      <Banner
+        image={'/images/samahan-members.png'}
+        imageAlt={'SAMAHAN Members'}
+        subtitle={'SAMAHAN'}
+        title={'GUIDESITE'}
+      />
+      <div className="text-blue font-sans px-8 md:px-32 py-10 bg-gray-100">
+        <div className="max-w-screen-lg mx-auto">
+          <section>
+            <h2 className="text-bold font-bold mb-4 pt-20">{title}</h2>
+
+            <ol className="list-decimal pl-5">
+              {steps.map((step) => (
```

Comment:
Mejo mahirap na basahin ang code dito kasi very nested na. Possible ba na maka gawa ng reusable component for steps then inside that component you can check kung ano ang pwede pa ma divide into reusable components